CONTRIBUTING.md

```diff
@@ -17,6 +17,10 @@ Jdbi places serious emphasis on not breaking compatibility. Remember these simpl
 
 If you must make some internal code `public` to access it from other packages, put the class in a package named `internal`. Packages named so are not considered API.
 
+### Forward compatibility
+
+*All* new API, with few exceptions, is to be marked with `@Beta`. This lets users know not to rely too much on your changes yet, as the public release might reveal that more work needs to be done.
```

Comment:
I think this statement might be too strong.  Large new features should strongly consider being `@Beta`.  But e.g. adding a new accessor to an existing class, to support a clearly positive design improvement, may not make as much sense.

Reply:
this better?